util/main.go

```diff
@@ -14,25 +16,36 @@ type SubmissionResults struct {
 
 // Record of a student's submission, with metadata about how it ran and compiled.
 type SubmissionResult struct {
-	Student        string
-	CompileSuccess bool
-	RunCorrect     bool
-	Feedback       string
-	AssignmentId   int8
-	StudentId      int8
+	Student        	string
+	CompileSuccess 	bool
+	Score						int8
```

Comment:
go fmt? looks odd in gh

Reply:
seems some other types did this too

Reply:
yeah no trailing newline so need to run that

Reply:
👍🏻